kratos/linear_solvers/iterative_solver.h

```diff
@@ -294,7 +294,7 @@ class IterativeSolver : public LinearSolver<TSparseSpaceType, TDenseSpaceType, T
         mIterationsNumber = NewIterationNumber;
     }
 
-    virtual unsigned int GetIterationsNumber()
+    virtual unsigned int GetIterationsNumber() override
```

Comment:
```suggestion
    unsigned int GetIterationsNumber() override
```

Reply:
virtual is only needed once in the baseclass, having it here doesn't do anything

Reply:
You mean if I derive a SpecialSolver from IterativeSolver, compiler considers this function as virtual since it is originally virtual in the base of the base (LinearSolver) class?

Reply:
yes exactly

check other classes, e.g. `Element::GetDofList`. The `virtual` is only needed once in the baseclass, the derived classes don't have it (they have `override` instead)

Reply:
Done.